src/plugins/intel_cpu/src/nodes/kernels/jit_kernel_traits.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <dnnl_types.h>
+#include <dnnl_extension_utils.h>
+
+namespace ov {
+namespace intel_cpu {
+
+using namespace dnnl::impl;
+using namespace dnnl::impl::utils;
+namespace x64 = dnnl::impl::cpu::x64;
+
+template<x64::cpu_isa_t isa, ov::element::Type_t element_type>
+class jit_kernel_traits {
```

Comment:
Could you add some usage examples?

Reply:
This file is no longer a part of this PR